lib/builders/List.js

```diff
@@ -302,6 +303,7 @@ function applySelection (parent, definition) {
   } else if (selectionType === 'single') {
     option.addAttribute(':val', rowKey ? `item.${rowKey}` : 'item')
     option.addAttribute(':value', `${destPath}`)
+    option.addAttribute(':aria-label', stringTemplateParser(definition.templates.label))
```

Comment:
Should this be applied outside of the `else if (selectionType === 'single')` block... so it is applied whether or not it's radio/checkbox ?

Reply:
Changes made